modules/rosa-upgrade-cluster-cli.adoc

```diff
@@ -0,0 +1,105 @@
+:_module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-manage-objects-cli.adoc
+
+[id="rosa-upgrade-cluster_{context}"]
+= Upgrade and delete upgrade for clusters
+
+[role="_abstract"]
+This section describes the `upgrade` command usage for clusters.
+
+== upgrade cluster
+
+Schedules a cluster upgrade.
+
+.Syntax
+[source,terminal]
+----
+$ rosa upgrade cluster [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster that the upgrade will be cancelled for.
+
+|--interactive
+|Enables interactive mode.
+
+|--version
+|The version (string) of OpenShift that the cluster will be upgraded to.
+
+|--schedule-date
+|The next date (string) the upgrade should run at the specified time. Format: `yyyy-mm-dd`
+
+|--schedule-time
+|The next time the upgrade should run on the specified date. Format:`HH:mm`
+
+|--node-drain-grace-period
+|Sets a grace period (string) for how long Pod Disruption Budget-protected workloads will be respected during upgrades. After this grace period, any workloads protected by Pod Disruption Budgets that have not been successfully drained from a node will be forcibly evicted. Default: `1 hour`
```

Comment:
s/Pod Disruption Budget/pod disruption budget
s/will be/are

Reply:
Done.